src/index.html

```diff
@@ -12,7 +13,61 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <!-- #endregion -->
+  <body class="body">
+    <div class="stars stars--0">
+      <ul>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
+    </div>
+    <div class="stars stars--1">
+      <ul>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
+    </div>
+    <div class="stars stars--2">
+      <ul>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
+    </div>
+    <div class="stars stars--3">
+      <ul>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
+    </div>
```

Comment:
```suggestion
      <ul class="stars stars--3">
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
      </ul>

```
looks like a redundant div